.github/workflows/unit_test_non_evalengine_mysql80.yml

```diff
@@ -0,0 +1,169 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
+
+name: Unit Test (non_evalengine_mysql80)
+on: [push, pull_request]
+concurrency:
+  group: format('{0}-{1}', ${{ github.ref }}, 'Unit Test (non_evalengine_mysql80)')
+  cancel-in-progress: true
+
+permissions: read-all
+
+env:
+  LAUNCHABLE_ORGANIZATION: "vitess"
+  LAUNCHABLE_WORKSPACE: "vitess-app"
+  GITHUB_PR_HEAD_SHA: "${{ github.event.pull_request.head.sha }}"
+
+jobs:
+  test:
+    name: Unit Test (non_evalengine_mysql80)
```

Comment:
```suggestion
    name: Unit Test (mysql80)
```

Reply:
I realize you need to change workflow gen to make these changes happen. It was easiest to suggest changes this way than to ci_workflow_gen.go

Reply:
As a bonus, keeping workflow names same as before will mean that you don't have "3 expected workflows" on this PR which don't exist any more.